lib/hoststats/cpu_test.go

```diff
@@ -0,0 +1,60 @@
+package hoststats
+
+import (
+	"math"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/consul/sdk/testutil"
+	"github.com/shirou/gopsutil/v3/cpu"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHostStats_CPU(t *testing.T) {
```

Comment:
I'm not sure of a better approach, but I don't like the thought of this being a unit test. We're essentially spinning a CPU core to generate some load, and this took almost 13s to run on my local machine.

Maybe we need a whole new integration test suite for this kind of stuff?